pkg/translator/prometheusremotewrite/go.mod

```
@@ -15,8 +15,11 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/kr/pretty v0.3.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.47.0 // indirect
```

```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.48.0 // indirect
```

Apologies, this slid into yet another release window.  Approving pending this suggestion so hopefully that doesn't happen again.